subprojects/settings.gradle.kts

```diff
@@ -122,10 +122,29 @@ include(":ci:k8s-deployments-cleaner")
 
 pluginManagement {
 
+    val artifactoryUrl: String? by settings
+
+    fun MavenArtifactRepository.artifactoryUrl(repositoryName: String) {
+        setUrl("$artifactoryUrl/$repositoryName")
+        isAllowInsecureProtocol = true
+    }
+
+    fun MavenArtifactRepository.setUrlOrProxy(repositoryName: String, originalRepo: String) {
+        if (artifactoryUrl.isNullOrBlank()) {
+            name = repositoryName
+            setUrl(originalRepo)
+        } else {
+            name = "Proxy for $repositoryName: $originalRepo"
+            artifactoryUrl(repositoryName)
+        }
+    }
```

Comment:
Is there any way to reuse these functions?

Reply:
NOPE :(